lang/contexts.json

```diff
@@ -1,6 +1,7 @@
 {
 	"Paragraph": "Drop-down option label for the paragraph format.",
-	"Heading": "Tooltip and default label for the heading drop-down.",
+	"Heading": "Tooltip for the heading drop-down.",
```

Comment:
>Do we need the tooltip at all?

> I'm for no label. Why was the label introduced?

I'm confused. Do you want the label or the tooltip gone? Tooltips have been introduced for consistency, that's all. See https://github.com/ckeditor/ckeditor5-heading/issues/55.